scripts/set-var.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"flag"
+	"time"
+
+	"github.com/mongodb/grip"
+	mgo "gopkg.in/mgo.v2"
+	"gopkg.in/mgo.v2/bson"
+)
+
+func main() {
+	var (
+		dbName     string
+		collection string
+		id         string
+		key        string
+		value      string
+	)
+
+	flag.StringVar(&dbName, "dbName", "mci_smoke", "database name for directory")
+	flag.StringVar(&collection, "collection", "", "name of collection")
+	flag.StringVar(&id, "id", "", "_id of document")
+	flag.StringVar(&key, "key", "", "key to set")
+	flag.StringVar(&value, "value", "", "value of key")
+	flag.Parse()
+
+	session, err := mgo.DialWithTimeout("mongodb://localhost:27017", 2*time.Second)
```

Comment:
does it make sense to use evergreen's session management here?

Reply:
I just copied set-project-var.go.